app/controllers/api/v1/courses_controller.rb

```diff
@@ -18,21 +19,26 @@ def index
                                  :subjects,
                                  site_statuses: [:site])
                        .changed_since(changed_since)
-                       .by_recruitment_cycle(recruitment_year)
+                       .by_recruitment_cycle(@recruitment_cycle.year)
                        .limit(per_page)
         end
 
         set_next_link_header_using_changed_since_or_last_object(
           @courses.last,
           changed_since: changed_since,
           per_page: per_page,
-          recruitment_year: recruitment_year
```

Comment:
Dropping this here makes me nervous, even though it feels like it *should* work, and there is another PR coming that should fix this. Can we actually make one PR for all the course changes, and another for all the provider changes.

Reply:
Now part of this PR. See below.